charter.html

```diff
@@ -0,0 +1,342 @@
+    </p>
+    <h2 id="meetings">
+      Meetings
+    </h2>
+    <p>
+      Meetings will be held biweekly on {TODO: INSERT DATE}, lasting for one
+      hour. Meetings will be run by one of the Chairs, who will request scribes
+      to take meeting notes that will be made publicly available in the
+      <a href="https://github.com/antifraudcg/meetings">Meetings</a> repository
+       after the meeting. Any action items from the meeting will also be posted
+       to the Community Group mailing list to bring them to the attention of
+       people who did not attend.
+    </p>
+    <p>
+      Infrequently, face-to-face meetings will be held for the Anti-Fraud
+      Community Group, allowing for more collaboration and iteration between
+      Community Group members.
+    </p>
+    <h2 id="deliverables">
+      Deliverables
+    </h2>
+    <h3 id="work-items">
+      Current Work Items
+    </h3>
+    <p>
+      There are currently no work items that have been adopted by the Community
+      Group. See <a href="#proposals">Proposals</a> for the process of adopting
+      a proposal into the Community Group.
+    </p>
+    <p>
+      Once a work item is ready for standardization and there is
+      <a href="#decision">consensus</a> that the work item is ready to
+      be migrated to a standards body, the Chairs will collaborate with the
+      Community Group to prepare a Community Group Report on the item, and work
+      with the Editors to disseminate it to the appropriate standards body.
+    </p>
+    <h3 id="proposals">
+      Proposals
+    </h3>
+    <p>
+      Anyone can submit proposals to the
+      <a href="https://github.com/antifraudcg/proposals/">proposals repository</a>.
+      If there is substantial documentation/work, the proposal authors can
+      request the Chairs set-up a dedicated repository for a proposal.
+    </p>
+    <p>
+      A proposal can be withdrawn by its authors, or closed if there's
+      consensus that the proposal is out of scope or the number of authors has
+      dropped to zero. The Chairs will check-in with active proposals to ensure
+      the set of active authors is kept up-to-date.
+    </p>
+    <p>
+      Generally proposals should include an <i>Explainer</i> document describing
+      the changes to the web platform and impact of the proposal.
+    </p>
+    <p>
+      Work items are proposals that have community group support and result in
+      a deliverable, either as interoperable standards/APIs or documentation.
+      Once a proposal is in a state for the community group to work on, it
+      should be proposed for adoption as a work item. To be adopted as a work
+      item, a proposal should be sent out to the CG mailing list, and there
+      must be at least two supporters of the proposal. For work items intended
+      to become a web-exposed API, at least one supporter should be a browser
+      vendor (as an indication of interest in implementation).
+    </p>
```

Comment:
At this point a state diagram might be helpful, or a comprehensive labeling of states (e.g., in a definition list). The charter refers to a number of states:
* New proposal
* Withdrawn proposal
* Closed proposal
* Proposed work item
* Adopted work item

I assume there would also be something like a "closed work item"; the charter does not speak to that (and I think it's fine that it not do so until we have more experience).

In short, it would be helpful to describe the states and processes for entering them.